modules/rostests/kmtests/ntos_ke/KeDpc.c

```diff
@@ -153,8 +162,10 @@ START_TEST(KeDpc)
     } _SEH2_END;
     ok_eq_hex(Status, STATUS_SUCCESS);
 
-    if (!skip(Status == STATUS_SUCCESS, "KeInitializeDpc failed\n"))
+    if (!skip(Status == STATUS_SUCCESS, "KeInitializeDpc failed\n") &&
+        GetCurrentNTVersion() < _WIN32_WINNT_WIN8)
     {
+        // Inserting NULL in a DPC gives a 0xC7 bugcheck on Windows 8+.
```

Comment:
nit so that the code reader can know what bsod 0xc7 is.
```suggestion
        // Inserting NULL in a DPC gives a 0xC7 (TIMER_OR_DPC_INVALID) bugcheck on Windows 8+.
```